libindy/src/api/mod.rs

```diff
@@ -141,6 +141,9 @@ pub enum ErrorCode
     // Timeout for action
     PoolLedgerTimeout = 307,
 
+    // Timeout for action
```

Comment:
1. Inconsistent description
1. Should be add to all libindy consumers: wrappers, plugin, cli